test/integration/apply_joint_force_system.cc

```diff
@@ -32,22 +32,16 @@
 #include "ignition/gazebo/test_config.hh"
 
 #include "../helpers/Relay.hh"
+#include "../helpers/EnvTestFixture.hh"
 
 #define TOL 1e-4
 
 using namespace ignition;
 using namespace gazebo;
 
 /// \brief Test fixture for ApplyJointForce system
-class ApplyJointForceTestFixture : public ::testing::Test
+class ApplyJointForceTestFixture : public InternalFixture<::testing::Test>
 {
-  // Documentation inherited
-  protected: void SetUp() override
-  {
-    ignition::common::Console::SetVerbosity(4);
-    ignition::common::setenv("IGN_GAZEBO_SYSTEM_PLUGIN_PATH",
-           (std::string(PROJECT_BINARY_PATH) + "/lib").c_str());
-  }
```

Comment:
I've noticed that this file (and many other files) have their `SetUp` method removed, instead of calling `InternalFixture::Setup`. I have also noticed that some files call `InternalFixture::SetUp`. What is the criteria for determining whether a file should have `SetUp` removed completely, or replaced with `InternalFixture::SetUp`?

Reply:
Since the tests now inherit from `InternalFixture`, they should be calling that class's `SetUp` by default. All setups that did what's now handled by the base class have been removed in favor of using the base class. 

`SetUp`s that do more than what the base class does override the function and call the base class's `SetUp` to make sure that runs as well.

Reply:
That makes sense :+1: sorry, I totally forgot about the override/inheritance behavior when reviewing this yesterday :upside_down_face: